linode/linode_api4-python
# Configure default Python version using integration-tests job environment

## 📝 Description

This pull request fixes a small issue that caused runs triggered by pushes to dev and main to not resolve a valid Python version, leading to a [failure](https://github.com/linode/linode_api4-python/actions/runs/11727401379/job/32870109952#step:4:1). This works by moving the default and EOL Python versions to the job-level environment rather than in the `workflow_dispatch` input defaults.

Sample Runs:
* [When triggered by a push](https://github.com/lgarber-akamai/linode_api4-python/actions/runs/11800462204)
* [When manually triggered using a custom Python version](https://github.com/lgarber-akamai/linode_api4-python/actions/runs/11800619197/job/32871951839)
* [When manually triggered with `Run EOL python version?` set to true](https://github.com/lgarber-akamai/linode_api4-python/actions/runs/11800620802/job/32871956758)

## ✔️ How to Test

### Manual Testing

1. Checkout this PR and push it up to your fork's main or dev branch.
2. Ensure the integration tests workflow successfully installs Python in the following cases:
    * When triggered by a push
    * When manually triggered using a custom Python version
    * When manually triggered with `Run EOL python version?` set to true
